tokio-trace/src/lib.rs

```diff
@@ -87,13 +87,23 @@
 //! //  - "foo", with a value of 42,
 //! //  - "bar", with the value "false"
 //! //  - "baz", with no initial value
-//! let my_span = span!(Level::INFO, "my_span", foo = 42, bar = false, baz);
+//! let my_span = span!(Level::INFO, "my_span", foo = 42, bar = false, baz = _);
 //!
 //! // record a value for the field "baz" declared above:
 //! my_span.record("baz", &"hello world");
 //! # }
 //!```
-//!
+//! As shorthand, local variables may be used as field values without an
+//! assignment, similar to [struct initializers]. For example:
+//! ```
+//! # #[macro_use]
+//! # extern crate tokio_trace;
+//! # use tokio_trace::Level;
+//! # fn main() {
+//! let user = "ferris";
+//! span!(Level::TRACE, "login", user);
```

Comment:
It would be good to show the same example with and without the shorthand.

Reply:
Good call! Will do.